modules/apps/callbacks/ibc_middleware_test.go

```diff
@@ -272,10 +276,10 @@ func (s *CallbacksTestSuite) TestOnAcknowledgementPacket() {
 			s.SetupTransferTest()
 
 			// set user gas limit above panic level in mock contract keeper
-			userGasLimit := 600000
+			userGasLimit = 600000
```

Comment:
does this get modified? Could probably be defined at the top of the test

Reply:
No, but the malleate needs access to this variable if it needs to change the memo. If you define it at the top, then an unexpected behavior can occur in which the modification of this variable from a test case can leak to the next test case